### ISSUE TITLE
SQL Database

### ISSUE THREAD

**Issue by MrBaileyQ** [Minehut/Meta#734]
### Feature request

**Checklist**
- [ ] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [ ] This is not a bug report, support request or plugin request
- [ ] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [ ] This issue does not contain any sensitive information 

**Platform**
In-Game

**Describe the feature**
I think there should be an option to create 1 MySQL database per payed server.
You should be able to manage it via the minehut website and be provided with the details

**How the feature is useful**
This feature will be useful to have when server networks release.
Example:
 You could be running a plugin that gives cosmetics. But you wanna have the same cosmetics on every server.
 You can link the plugin with the mySQL database, so the player's keep their items

This would also be useful for custom plugins people make that go on each server.
If you wanted to store information such as variables eg: level, nickname ect.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by Reimagining** [comment]
I mean, many other hosts offer this with plans and minehut is a premium, but the up costs to even build the Database hardware was discussed to be expensive, so I do not think this would happen as an included feature (maybe an add-on).

**Comment by tarna** [comment]
Trent has talked about having a feature where Minehut hosts SQL or even a Mongo database for people to use on their servers but there was just never an actual issue made about it. This seems like a cool idea and I also don't think it should be something that is available on the free plan. Each paid server plan could have a higher storage limit on these databases or something.

**Comment by Reimagining** [comment]
Yeah, as most other hosts offer them as a standard, it would be nice for minehut to offer it, especially with their current pricing.
